benchmark-framework/src/main/java/io/openmessaging/benchmark/Benchmark.java

```diff
@@ -31,158 +21,196 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.ObjectWriter;
 import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
-
 import io.openmessaging.benchmark.worker.DistributedWorkersEnsemble;
 import io.openmessaging.benchmark.worker.LocalWorker;
 import io.openmessaging.benchmark.worker.Worker;
+import java.io.File;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class Benchmark {
 
-    static class Arguments {
-
-        @Parameter(names = {"-c", "--csv"}, description = "Print results from this directory to a csv file")
-        String resultsDir;
-
-        @Parameter(names = { "-h", "--help" }, description = "Help message", help = true)
-        boolean help;
-
-        @Parameter(names = { "-d",
-                "--drivers" }, description = "Drivers list. eg.: pulsar/pulsar.yaml,kafka/kafka.yaml")//, required = true)
-        public List<String> drivers;
-
-        @Parameter(names = { "-w",
-                "--workers" }, description = "List of worker nodes. eg: http://1.2.3.4:8080,http://4.5.6.7:8080")
-        public List<String> workers;
-
-        @Parameter(names = { "-wf",
-                "--workers-file" }, description = "Path to a YAML file containing the list of workers addresses")
-        public File workersFile;
+  static class Arguments {
+
+    @Parameter(
+        names = {"-c", "--csv"},
+        description = "Print results from this directory to a csv file")
+    String resultsDir;
+
+    @Parameter(
+        names = {"-h", "--help"},
+        description = "Help message",
+        help = true)
+    boolean help;
+
+    @Parameter(
+        names = {"-d", "--drivers"},
+        description =
+            "Drivers list. eg.: pulsar/pulsar.yaml,kafka/kafka.yaml") // , required = true)
+    public List<String> drivers;
+
+    @Parameter(
+        names = {"-w", "--workers"},
+        description = "List of worker nodes. eg: http://1.2.3.4:8080,http://4.5.6.7:8080")
+    public List<String> workers;
+
+    @Parameter(
+        names = {"-wf", "--workers-file"},
+        description = "Path to a YAML file containing the list of workers addresses")
+    public File workersFile;
+
+    @Parameter(
+        names = {"-x", "--extra"},
+        description = "Allocate extra consumer workers when your backlog builds.")
+    boolean extraConsumers;
+
+    @Parameter(description = "Workloads") // , required = true)
+    public List<String> workloads;
+
+    @Parameter(
+        names = {"-o", "--output"},
+        description = "Output",
+        required = false)
+    public String output;
+  }
+
+  public static void main(String[] args) throws Exception {
+    final Arguments arguments = new Arguments();
```

Comment:
Probably we should keep 4 spaces indent?

Reply:
We've configured the `googleJavaFormat` which is 2 spaces and by design it's not customisable. There is the AOSP (Android) flavour of the Google Java format, which is 4 spaces.

Reply:
We can just add it to the java format conf:
```
<java>
            <googleJavaFormat/>
            <importOrder/>
            <removeUnusedImports/>
            <indent>
              <tabs>true</tabs>
              <spacesPerTab>2</spacesPerTab>
            </indent>
            <indent>
              <spaces>true</spaces>
              <spacesPerTab>4</spacesPerTab>
            </indent>
          </java>```


This way we won't have to reformat 100% of the code, since it's already mostly with 4 spaces.